continuous_integration/azure/install.bat

```diff
@@ -34,3 +34,7 @@ if errorlevel 1 exit 1
 :: Install codecov
 pip install -q codecov
 if errorlevel 1 exit 1
+
+:: Install pywinpty
+pip install -q pywinpty
```

Comment:
This is not correct. Please remove it from here and add it instead to `requirements/conda.txt`

Reply:
Is it possible to mark ``pywinpty`` as a OS-dependent dependency?

Reply:
Sorry, you're right. Then please add a new file called `requirements/windows.txt` and use it only on Windows.

Besides, you need to update our Contributing guide to mention that. And also our Readme to mention the new dependency.